tests/integration/tools.js

```diff
@@ -37,4 +37,24 @@ export function clearStore(page) {
   page.evaluate(() =>
     localStorage.clear()
   )
-}
\ No newline at end of file
+}
+
+export async function toggleFavoritePanel(page) {
+  await page.waitForSelector('.icon-icon_star.side_bar__item__icon')
+  await page.click('.icon-icon_star.side_bar__item__icon')
+  await wait(300)
+}
+
+// Returns all favorites as an array
+// It needs the favorite panel to be open
+export async function getFavorites(page) {
```

Comment:
Maybe it's too soon but we should abstract some helper by context (Fav, Poi etc) 
This one is too specific to fit inside "tool"

Reply:
so you want me to move this function to a `favorite_tools.js` ?